superset/db_engine_specs.py

```diff
@@ -590,6 +590,58 @@ def get_table_names(cls, inspector, schema):
         """Need to disregard the schema for Sqlite"""
         return sorted(inspector.get_table_names())
 
+class DrillEngineSpec(BaseEngineSpec):
+    """Engine spec for Apache Drill"""
+    engine = 'drill'
+
+    @classmethod
+    def extra_table_metadata(cls, database, table_name, schema_name):
+        """Returns engine-specific table metadata"""
+
+        return {}
```

Comment:
I doin't think you need to override as the base class does the same

Reply:
Fixed